### DISCUSSION TITLE
Profile Not visible in Google searchs

### DISCUSSION THREAD

**Discussion by HoudaifaRbahi** [community/community#140210] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi Github Support
I'm facing a probleù that my github account is not visible when taping the account Name in google search

**Comment by 7x9z** [comment]
Hello, @HoudaifaRbahi,
You do in fact appear in "Google searches"
![image](https://github.com/user-attachments/assets/e6afca3f-04f4-41c0-86bf-f79948738cbb)
![image](https://github.com/user-attachments/assets/65e94499-1784-4ae3-9d07-c561766f28d5)

**Comment by moranenzo** [comment]
<p>Hi there!</p>

<p>It can be frustrating when your GitHub profile isn’t showing up in Google searches. Sometimes, it just takes a little while for search engines to index new or updated profiles. You can also try checking your profile’s visibility settings on GitHub to make sure it’s public. If the issue persists, reaching out to Google Support or waiting a bit longer may help!</p>

<p>Hope this helps, and best of luck!</p>

**Comment by YasinDehfuli** [comment]
I have the same issue as you on my repository.

one of my open-source repositories. 

https://github,com/4xmen/get-github-achivements

didn't show in Google or any other search engine response.

<img width="913" alt="image" src="https://github.com/user-attachments/assets/ab9c5d1a-da44-4120-bc1a-b07e41988b68">

It shows all the other pages of my project, but no direct link to my project exists.

**Reply by 7x9z** [reply]
Hey, @HoudaifaRbahi The repo link you have gave seems to not exist or is private,this is a reason why it does not show up in search's.
Hope this helps!
